src/PekatVisionSDK/instance.py

```diff
@@ -375,6 +375,36 @@ def analyze(
 
         raise InvalidDataTypeError(type(image))
 
+    def send_random(
+        self,
+        shape: Tuple[int, ...] = (512, 512, 3),
+        response_type: ResponseType = "context",
+        data: Optional[str] = None,
+        timeout: float = 20,
+    ) -> Result:
+        """Send random data for analysis.
+
+        Arguments:
+            shape: Shape of the image to be sent.
+            response_type: Type of response data.
+            data: Data to be added to the query.
+                Project will be able to access this under the `"data"` key in `context`.
+            timeout: Timeout in seconds for the analyze request.
+
+        Raises:
+            Exception: Same as [`Instance.analyze`][PekatVisionSDK.Instance.analyze]
+
+        Returns:
+            A tuple of image and `context`.
+                If `response_type` is `"context"`, then the returned image is `None`.
+        """
+        return self.analyze(
+            (np.random.default_rng().random(shape) * 256).astype(np.uint8),
```

Comment:
Is there some advantage in using the "random" method, multiplication and type change, instead of directly generating the uint8 integers in the specified 0-255 range with the "integers" method? https://numpy.org/doc/stable/reference/random/generated/numpy.random.Generator.integers.html#numpy.random.Generator.integers

Reply:
No advantage, I haven't realized numpy has this functionality, thank you!